Apps/Playground/UWP/App.cpp

```diff
@@ -208,26 +208,28 @@ void App::OnWindowClosed(CoreWindow^ sender, CoreWindowEventArgs^ args)
 
 void App::OnPointerMoved(CoreWindow^, PointerEventArgs^ args)
 {
-    if (m_inputBuffer != nullptr)
+    if (nativeInput)
```

Comment:
nit: I would recommend doing this for raw pointers:
```suggestion
    if (nativeInput != nullptr)
```

Reply:
Fixed